### DISCUSSION TITLE
Is there a way to update a  library to latest version as Deno caches packages from a CDN?

### DISCUSSION THREAD

**Discussion by murunwas** [windmill-labs/windmill#1602] [Questions]
I have imported a package from private registry `import {greet} from "https://private.example.com/mod.ts` 
when I push new changes, the script can't get the latest changes, is there a way to force script to pull latest changes??

**Comment by rubenfiszel** [comment]
You can either nuke your cache or use fixed version. Using non pinned version is really non recommended
